File: test/js/resilience/broken.js
// Deliberately malformed JavaScript (unbalanced parens/braces) used only to
// prove that converters/javascript's directory walk skips an unparseable
// file rather than aborting the whole batch conversion. See
// converters/javascript/converter_test.go's
// TestConvertDirectorySkipsUnparseableFile.
function totallyBroken( {
  return 1;

File: test/python/resilience/broken.py
# Deliberately malformed Python (unbalanced parens, invalid syntax) used only
# to prove that converters/python's directory walk skips an unparseable file
# rather than aborting the whole batch conversion. See
# converters/python/converter_test.go's
# TestConvertFile_DirectorySkipsUnparseableFile.
def totally_broken(:
    return 1
